crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/replace_bytecode.rs

```diff
@@ -11,3 +13,15 @@ impl CheatnetState {
             .insert(contract_address, class_hash);
     }
 }
+
+pub enum ReplaceBytecodeError {
+    ContractNotDeployed,
+}
+
+impl SerializeAsFelt252Vec for ReplaceBytecodeError {
```

Comment:
We could have a derive macro for that 🤔 cc @Draggu 
(really low prio though)

Reply:
I want to move this and `FromReader` into `conversions`, rename to `CairoSerde` and add this macro. For now it's good @piotmag769

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/2109

Reply:
@Draggu  if you want to add some details, please do in the issue above 🙏

Reply:
@Arcticae as @piotmag769 stated it's low prio

Reply:
@Draggu yea, it is more about including short implementation idea so that the person working on this will know what we had in mind